demisto_sdk/commands/common/hook_validations/id.py

```diff
@@ -397,3 +400,37 @@ def is_file_valid_in_set(self, file_path, file_type, ignored_errors=None):
                 is_valid = self._are_playbook_entities_versions_valid(playbook_data, file_path)
 
         return is_valid
+
+    def _is_pack_display_name_already_exist(self, pack_metadata_data):
+        """Check if the pack display name already exists in our repo
+        Args:
+            pack_metadata_data (dict): Dictionary that holds the extracted details from the given metadata file.
+        Returns:
+            bool. Whether the metadata file is valid or not.
+        """
+        new_pack_folder_name = list(pack_metadata_data.keys())[0]
+        new_pack_name = pack_metadata_data[new_pack_folder_name]['name']
+        for pack_folder_name, pack_data in self.packs_set.items():
+            if new_pack_name == pack_data['name'] and new_pack_folder_name != pack_folder_name:
+                return False, Errors.pack_name_already_exists(new_pack_name)
+        return True, None
+
+    def is_unique_file_valid_in_set(self, pack_path, ignored_errors=None):
```

Comment:
Unique how? Valid in what way? Is it a file or necessarily a pack?
This name is a bit confusing to me, I suggest reconsidering it

Reply:
Regular files - 
![image](https://user-images.githubusercontent.com/57212002/109479581-31d0bd00-7a83-11eb-998b-26922a21cd56.png)
Unique files - 
![image](https://user-images.githubusercontent.com/57212002/109479617-39906180-7a83-11eb-84c9-21c5704d64f4.png)
We have `is_file_valid_in_set` function for the regular files
and I added `is_unique_file_valid_in_set ` function